test/helpers/kubectl.go

```diff
@@ -618,6 +671,98 @@ func (kub *Kubectl) CiliumPolicyAction(namespace, filepath string, action Resour
 	return "", nil
 }
 
+// CiliumExportInfo runs a subroutine in the background, that exports the
```

Comment:
the Cilium --> Cilium's
prometheus --> Prometheus.
a context to stop the function when is needed and prefix --> a context to stop the function when needed and a prefix
gather --> gathers
each --> every